### PR TITLE
M3-5661: Add Databases and Firewalls to Support Tickets

### PR DESCRIPTION
**Pull request by bnussman**
## Description

- Allows users to link **Databases** or **Firewalls** in a support ticket

## How to test

- Point Cloud to the alpha environment 
- Have a Database on your account
- Go to create a support ticket and link your database to the ticket
- Verify it was linked in alpha

Repeat the steps above for other entities such as Linodes and Firewalls to ensure I didn't break anything .

### CODE REVIEW

File: packages/manager/src/features/Support/SupportTickets/SupportTicketDrawer.tsx [hunk 1]
```diff
@@ -443,8 +453,42 @@ export const SupportTicketDrawer: React.FC<CombinedProps> = (props) => {
...
+    if (entityType === 'firewall_id') {
+      return (
+        firewalls?.map((firewall) => ({
+          value: firewall,
+          label: firewall.label,
+        })) || []
+      );
+    }
+    return data;
+  };
```
Comment (bnussman) [inline packages/manager/src/features/Support/SupportTickets/SupportTicketDrawer.tsx:474]: Will try to clean this up... Any suggestions appreciated!

**Review by HanaXu — APPROVED** [review]

File: packages/manager/src/queries/databases.ts [hunk 2]
```diff
@@ -98,6 +105,11 @@ export const useDatabaseBackupsQuery = (engine: Engine, id: number) =>
     () => getDatabaseBackups(engine, id)
   );
 
+export const getAllDatabases = () =>
+  getAll<DatabaseInstance>((params) => getDatabases(params))().then(
+    (data) => data.data
+  );
+
```
Comment (codyfinn) [inline packages/manager/src/queries/databases.ts:112]: Could this be made async/await?

File: packages/manager/src/queries/firewalls.ts [hunk 3]
```diff
@@ -26,6 +26,11 @@ const getAllFirewallsRequest = () =>
     getFirewalls(passedParams, passedFilter)
   )().then((data) => listToItemsByID(data.data));
 
+const getAllPlainFirewallsRequest = () =>
+  getAll<Firewall>((passedParams, passedFilter) =>
+    getFirewalls(passedParams, passedFilter)
+  )().then((data) => data.data);
+
```
Comment (codyfinn) [inline packages/manager/src/queries/firewalls.ts:33]: Can this be made async/await?

**Review by codyfinn — APPROVED** [review]
Looks and works good! Just a couple of non-blocking comments.

**Pull request merged**